src/index.js

```diff
@@ -93,8 +98,9 @@ import sh from 'shelljs';
     } catch (err) {
       throw new Error(`Provided --project-dir \`${rawProjectDir}\` is not a directory.`);
     }
-    if (userModulePath.indexOf(`${projectDir}/`) !== 0) {
-      throw new Error(`File \`${userModulePath}\` must be located within --project-dir \`${rawProjectDir}\``);
+    if (userModulePath.indexOf(`${projectDir}/`) !== 0
+    && userModulePath.indexOf(`${projectDir}\\`) !== 0) {
```

Comment:
Let's use the `path` module [`sep` helper](https://nodejs.org/api/path.html#path_path_sep).

Also, we could make this clearer with `startsWith`, which has node 6 support.

```js
if (userModulePath.startsWith(`${projectDir}${path.sep}`) {
```

Reply:
Great point! I did not even know about `path.sep` – should have read Node.js basics before doing things 😅